.travis.yml

```diff
@@ -1,32 +1,21 @@
 ---
 language: python
-python:
-  - "2.7"
-
-sudo: required
 
 services:
   - docker
 
-cache:
-  pip: true
-  directories:
-    - $HOME/docker
+python:
+  - "2.7"
+  - "3.6"
 
-before_install:
-  - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
-  - curl -L https://github.com/docker/compose/releases/download/1.20.1/docker-compose-`uname -s`-`uname -m` > docker-compose
```

Comment:
maybe is posible to send the version (1.20.1 in this case) as a variable to avoid manual updates of this file?

Reply:
@speedlight sure but that is no longer necessary since we are not using `docker-compose` anymore, `molecule` is in charge of spinning up the docker containers by itself.

Reply:
oh sorry! confuse the colors!